venues/aclweb.org/NAACL/2022/special-theme/post_assignment_edges.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import openreview
+from tqdm import tqdm
+import csv
+from openreview import tools
+import re
+from collections import defaultdict
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+def assign_reviewer_to_paper(reviewer_id_or_email, paper_forum):
+    reviewer = client.get_profile(reviewer_id_or_email)
```

Comment:
if the reviewer does not have a profile, this will throw an error

Reply:
I think this is intended, so that they will know if there is a conflict